app/i18n/locales/en-US.json

```diff
@@ -177,7 +177,9 @@
   "wallet.trezor.dialog.trezor.save.button.label": "Save",
   "wallet.trezor.dialog.trezor.common.step.link.helpYoroiWithTrezor": "https://yoroi-wallet.com/",
   "wallet.trezor.dialog.trezor.common.step.link.helpYoroiWithTrezor.text": "Click here to know more about how to use Yoroi with Trezor",  
-  "wallet.trezor.dialog.trezor.step.about.introText": "Trezor hardware wallet is a small USB device that allows you to access your wallet quickly, safely & easily. It is more secure because your private key never leaves the hardware wallet. It protects you from phishing, malware, and more.",
+  "wallet.trezor.dialog.trezor.step.about.introText.line.1": "Hardware wallet is a small USB device that allows you to access your wallet quickly, safely & easily.",
```

Comment:
I would change it to:  ```A hardware wallet is a small USB device that allows you to give an extra level of security to your wallet.```

Reply:
revised by KP: `A hardware wallet is a small USB device that adds an extra level of security to your wallet.`